packages/app/src/components/Admin/Security/LocalSecuritySettingContents.jsx

```diff
@@ -186,13 +192,20 @@ class LocalSecuritySettingContents extends React.Component {
                     type="checkbox"
                     className="custom-control-input"
                     id="isEmailAuthenticationEnabled"
+                    disabled={!isMailerSetup}
```

Comment:
switch の disabled はいらない気がする。
この機能が ON になったあと、isMailerSetup が false になったら、機能を OFF にすることができない(動かないままエラーが出続ける)

## To-be
- ゴールを「SMTP設定が完了していない場合、メール認証を有効にできないようにする 」から「SMTP設定が完了していない場合、メール認証が働かないようにする」に変える
    - 実際に働かないようにするコードを入れる
- LocalSecuritySettingContents.jsx では、switch は ON/OFF 自由にできてよい
    - alert-warning がでればそれでいいことにする

Reply:
Maybe you don't need the `disable` for the switch.
Once this function is turned on and `isMailerSetup` becomes `false`, you will never be able to turn it off. (It will stop running, and just continue to throw errors)

# To-be
* Change your goal from "A user should not be able to enable email auth if the configuration for the SMTP settings has not been completed" to "Email auth should not work if the configuration for the SMTP settings has not been completed"
  * Add code so that email auth does not work for this case.
* The switch should become freely ON/OFF at  `LocalSecuritySettingContents.jsx`.
  * Just showing `alert-warning` is enough for now.

# Next Action
* Review by @kaoritokashiki

Reply:
# Review
## TODO in this task
1. please do not apply `disabled` to the switch button.
2. please add `alert-danger` like the following image when the email settings are not completed.
-> alert message(ja): 
「以下の機能を使えるようにするには、メール設定を完了させてください。」
please also add a link to jump to Mail Settings

<img width="984" alt="Screen Shot 2021-11-08 at 18 51 47" src="https://user-images.githubusercontent.com/59536731/140720852-6351e152-7eba-4bf9-a029-d425b36d5722.png">


## TODO in another task
- https://youtrack.weseek.co.jp/issue/GW-7577 (this task will be translated by @r-murakami )